src/PasswordResetController.js

```diff
@@ -45,6 +45,32 @@ function (Okta, FormController, FormType, ValidationUtil, FooterSignout, TextBox
     Form: {
       save: Okta.loc('password.reset', 'login'),
       title: Okta.loc('password.reset.title', 'login'),
+      subtitle: function () {
+        var policy = this.options.appState.get('policy');
+        if (policy.complexity) {
```

Comment:
Thanks. Done.

You are right, the `policy` is not guaranteed. Added additional condition.